qabelbox/src/androidTest/java/de/qabel/qabelbox/ui/files/FilesFragmentUITest.java

```diff
@@ -156,10 +158,10 @@ public void setUp() throws IOException, QblStorageException {
         wakeLock = UIActionHelper.wakeupDevice(mActivity);
         mSystemAnimations = new SystemAnimations(mActivity);
         mSystemAnimations.disableAll();
+        mActivity.runOnUiThread(() -> mActivity.selectIdentity(testIdentity));
```

Comment:
if you don't run the activity automatically, you don't need to use such a hack. Just select the activity service and then start the activity.